testpmd/hooks/user-tests.yml

```diff
@@ -3,3 +3,21 @@
   include_role:
     name: example-cnf-validate
   when: run_migration_test|default(true)|bool
+
+- name: Get logs from pods
+  community.kubernetes.k8s_log:
+    namespace: example-cnf
+    label_selectors:
```

Comment:
Is it possible to all the pods in this namespace instead of chosing by labels? The reason is some releases will have specific pods, but some will not. We can have the log collection generic to log all the pods and containers.